frameworks/kafka/client/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM confluentinc/cp-kafka:4.1.0
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        unzip
+
+ARG JAVA_TAR_ARCHIVE="jre-8u152-linux-x64-jce-unlimited.tar.gz"
```

Comment:
didn't all frameworks get bumped to `192`? or was that only for `0.50.x` SDK?

Reply:
The `client` Dockerfile was removed from master, but never from the sdk-0.40 branch.